lib/src/main/scala/spinal/lib/CrossClock.scala

```diff
@@ -6,9 +6,10 @@ import scala.collection.Seq
 import scala.collection.mutable.ArrayBuffer
 
 object BufferCC {
-  def apply[T <: Data](input: T, init: => T = null, bufferDepth: Option[Int] = None, randBoot : Boolean = false): T = {
-    val c = new BufferCC(input, init, bufferDepth, randBoot)
+  def apply[T <: Data](input: T, init: => T = null, bufferDepth: Option[Int] = None, randBoot : Boolean = false, inputAttributes: Seq[SpinalTag] = List(), allBufAttributes: Seq[SpinalTag] = List()): T = {
```

Comment:
Do we even need `allBufAttributes`? The only use-case I see is for false paths - and for those a single one is sufficient to break the timing arc that we want to prevent.

Reply:
Location used: https://github.com/SpinalHDL/SpinalHDL/pull/1369/files#diff-5e6bbb8e52385971285b669e6d58869c6b0c3f3e9fb488b8917e2a30931879d3R138-R143

Reply:
You are right, I was forgetting that we don't create any `-through` constraints, but `-from`/`-to`.